app/views/funding/trainee_summaries/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<%= render PageTitle::View.new(i18n_key: "funding") %>
+
+<%= content_for(:breadcrumbs) do %>
+  <%= render GovukComponent::BackLinkComponent.new(text: t(:back), href: root_path) %>
+<% end %>
+
+<% if current_user.organisation %>
+  <span class="govuk-caption-xl"><%= current_user.organisation.name %></span>
+<% end %>
+
+<h1 class="govuk-heading-xl">
+  <%= t("funding.view.title") %>
+</h1>
+
+<%= render TabNavigation::View.new(items: [
+  { name: t("funding.payment_schedule.title"), url: funding_monthly_payments_path },
```

Comment:
I think this is `funding_payment_schedule_path` in https://github.com/DFE-Digital/register-trainee-teachers/pull/2355